webpack.config.prod.js

```diff
@@ -17,7 +17,7 @@ const config = {
     fs: "empty"
   },
   mode: 'production',
-  devtool: 'source-map',
+  devtool: process.env.NODE_ENV === "development" ? "source-map" : "none",
```

Comment:
Since this is  only used in production, we can leave this as "source-map". According to dev tools it's safe to use source maps in production: https://webpack.js.org/configuration/devtool/

And this way we have more detailed information when encountering bugs.